src/main/scala/performanceanalysis/administrator/LogParserActor.scala

```diff
@@ -0,0 +1,31 @@
+package performanceanalysis.administrator
```

Comment:
Why did you put this in the administrator package? The actor will be communicating with both the AdministratorActor and the LogReceiverActor, so with respect to functional cohesion it could as well be put in the logreceiver package.